beacon-chain/core/helpers/committee.go

```diff
@@ -22,6 +22,11 @@ import (
 var committeeCache = cache.NewCommitteesCache()
 var proposerIndicesCache = cache.NewProposerIndicesCache()
 
+// ResetCommitteeCache is used from other packages (in tests) to reset the cache.
+func ResetCommitteeCache() {
```

Comment:
We also have `ClearCache()` Perhaps we should standardize the naming.

Reply:
Yep, Nishant suggested that and it is fixed in https://github.com/prysmaticlabs/prysm/pull/8706 (will rebase this branch once 8706 is merged)